content/pom.xml

```diff
@@ -74,6 +74,12 @@
                         <configuration>
                             <generateJavaClasses>true</generateJavaClasses>
                             <generatedJavaClassesPrefix>org.apache.sling.scripting.sightly</generatedJavaClassesPrefix>
+                            <allowedExpressionOptions>
```

Comment:
This is a new parameter for the `htl-maven-plugin`. For more context see [0] and [1].

[0] - https://sling.apache.org/components/htl-maven-plugin/validate-mojo.html#allowedExpressionOptions
[1] - https://issues.apache.org/jira/browse/SLING-6779

Reply:
I stumbled upon this PR when researching the following `htl-maven-plugin` warning:
```
[WARNING] /path/to/file.html [XX:XX]: ${'res' @ resourceType='resource/type', decoration=true}: Unknown option 'decoration'.
```

Just wanted to leave this information here for whoever will be searching for the same information.

Based on [this SLING ticket](https://issues.apache.org/jira/browse/SLING-6779), expression options like 'decoration' and 'wcmmode' are specific to AEM, and thus are not part of the core HTL spec. So, these options need to be defined as allowed in the plugin config.